instrumentation/lettuce/lettuce-5.1/library/src/main/java/io/opentelemetry/instrumentation/lettuce/v5_1/OpenTelemetryTracing.java

```diff
@@ -194,11 +193,22 @@ private OpenTelemetrySpan nextSpan(Context context) {
     @Override
     public synchronized Tracer.Span remoteEndpoint(Endpoint endpoint) {
       if (endpoint instanceof OpenTelemetryEndpoint) {
-        fillEndpoint(span, spanBuilder, (OpenTelemetryEndpoint) endpoint);
+        fillEndpoint((OpenTelemetryEndpoint) endpoint);
       }
       return this;
     }
 
+    private void fillEndpoint(OpenTelemetryEndpoint endpoint) {
+      AttributesBuilder attributesBuilder = Attributes.builder();
+      Context currentContext = span == null ? context : context.with(span);
+      netAttributesExtractor.onEnd(attributesBuilder, currentContext, endpoint, null, null);
```

Comment:
using `Context.root()` seems like it would be ok for this odd usage, but ok with this too